vignettes/report-writing-vignette.Rmd

```diff
@@ -0,0 +1,313 @@
+column should be named, *align* specifies how to align the data within each 
+column, *caption* specifies the title for the table, *booktabs* specifies 
+formatting for the table (i.e., the bolded column headers, the horizontal lines,
+the subtle horizontal lines between each row), and *format.args* specifies 
+format arguments to be used in the table. In this instance, *big.mark = ","* 
+specifies that commas should be inserted to the left of the decimal point when 
+necessary to improve readability (i.e., printing 3,400 instead of 3400).
+
+```{r iris, echo = TRUE, eval = FALSE}
+kable(x = temp, digits = 1, col.names = c("Sepal Length", "Sepal Width", 
+                                          "Petal Length", "Petal Width", 
+                                          "Species"), 
+      align = "c", caption = "A table of irises with long petals", 
+      booktabs = TRUE, format.args = list(big.mark = ","))
+```
+
+Tables may be referenced using the following inline command: 
+`\@ref(tab:chunk-label)`. Please note, however, that the text to the right of 
+the colon refers to the code chunk name. Of special importance, cross-references
+cannot use an underscore in the name of the code chunk. The underscore is a 
+special character in Latex, which causes difficulties in referencing table code 
+chunks using an underscore in the name.
+
+Although not shown above, the code chunk containing the table was labeled *iris*.
+In completing this example, we can then reference the numbered table using 
+`\@ref(tab:iris)`, and a hyperlink will appear with the number of the table. The 
+hyperlink can be clicked to direct the user to the location of the table in the 
+document.
+
+## Create and Cross Reference Figures
+
+Cross-referencing figures can be done similarly to cross-referencing tables. The
+most notable difference is the syntax for referencing a figure. To reference a 
+figure, we can use `\@ref(fig:chunk-label)`. Once again, *chunk-label* refers to
+the name of the code chunk containing the figure. In this case, the code chunk 
+name is *iris-figure*. Therefore, we can use `\@ref(fig:iris-figure)` to 
+reference the figure.
+
+A more in-depth introduction to ggplot can be found in the plotting vignette, 
+which can be called by typing `vignette(reporting-writing-vignette)`.
+
+## Using Inline R Code 
+
+Up to this point, all of the code that has been discussed has involved code 
+chunks. Code can also be evalued inline so that the results are presented in the
+sentence without special formatting. By using `` `r
+r_code` ``, the results of *r_code* would be pasted into the text. Consider the 
+following example. If I am describing the iris dataset, I might want to state 
+the mean petal length in the text. To do this using inline code, I might say:
+
+The mean petal length of the irises was `r mean(iris$Petal.Length)`, which uses
+`` `r
+mean(iris$Petal.Length)` `` as the inline coded. 
+
+## Combining Words
```

Comment:
I think this section and the next can be combined into a "tips and tricks" section that will probably grow as we do this more. Both can again be shorted. Just brief examples of "here's this thing you might want to do", "here's a brief example", and "here's a link where you can go to learn more."